.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,88 @@
+name: Bug report
+description: Create a bug report
+labels: [ bug ]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of the bug, including steps to reproduce it and the normally expected behavior.
+        You can also attach screenshots or screen recordings to help explain your issue.
+      placeholder: |
+        1. Go to …
+        2. Click on …
+        3. Scroll down to …
+        4. See error / the app crashes
+
+        Instead, I expect …
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: |
+        Please paste your client logs (logcat) here, *NOT* server logs (in most cases).
+        Learn how to capture those logcats [here](https://wiki.lineageos.org/logcat.html).
+      placeholder: Paste logs…
+      render: shell
+  - type: input
+    id: app-version
+    attributes:
+      label: Applicaiton version
+      description: The version of the installed Jellyfin Android app.
+      placeholder: 2.3.0
+    validations:
+      required: true
+  - type: dropdown
+    id: installation-source
+    attributes:
+      label: Where did you install the app from?
+      description: Choose the appropriate app store or installtion method.
+      options:
+        - Google Play
+        - F-Droid
+        - Amazon Appstore
+        - Sideloaded APK (GitHub, etc.)
+  - type: dropdown
+    id: app-variant
+    attributes:
+      label: Which variant are you using?
+      description: Proprietary (with cast support - Google Play, Amazon) or libre (F-Droid).
+      options:
+        - Proprietary
+        - Libre
+  - type: input
+    id: device-info
+    attributes:
+      label: Device information
+      description: Manufacturer, model
+      placeholder: Google Pixel 5, Samsung Galaxy S21
+    validations:
+      required: true
+  - type: input
+    id: android-version
+    attributes:
+      label: Android version
+      description: Version of the OS and other information (e.g. custom ROM / OEM skin)
+      placeholder: Android 11, Android 10
```

Maybe extend the placeholder to show an example for a custom ROM like this;
```suggestion
      placeholder: Android 11, LineageOS 10
```